frontend/src/data/settings.js

```diff
@@ -0,0 +1,6 @@
+import { createResource } from "frappe-ui"
+
+export const EnableCheckinForMobile = createResource({
```

Comment:
```suggestion
export const allowEmployeeCheckin = createResource({
```

Reply:
Use camelCase everywhere and fix formatting

run this in the frontend folder:

`npx prettier . --write`